server/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -1052,7 +1052,7 @@ public void testCreateTableGeneratedColumnWithCast() {
     @Test
     public void testCreateTableWithCurrentTimestampAsGeneratedColumnIsntNormalized() {
         BoundCreateTable analysis = analyze(
-            "create table foo (ts timestamp with time zone GENERATED ALWAYS as current_timestamp)");
+            "create table foo (ts timestamp with time zone GENERATED ALWAYS as current_timestamp(3))");
```

Comment:
This change kinda invalidates the comment further below: `assertThat(generatedColumnsMapping.get("ts"), is("current_timestamp(3)")); // 3 is the default precision`. Either the change should be reverted or the comment removed.